lib/pipelines/compileSoy.js

```diff
@@ -108,17 +108,27 @@ function extractParams() {
 			var parsed = getParsedSoy(soyJsPath, file.contents);
 
 			templateData[soyJsPath] = {};
-			parsed.templates.forEach(function(cmd) {
-				if (cmd.deltemplate) {
+
+			parsed.body.forEach(function(template) {
+				if (template.deltemplate) {
 					return;
 				}
 
-				var templateName = cmd.name;
+				var templateName = template.id.name;
```

Comment:
Just be aware that this will only be something like `.render`, and `namespace` is a separate attribute on `id`.